specification/dartLangSpec.tex

```diff
@@ -1986,7 +1986,8 @@ \subsubsection{Optional Formals}
 <defaultFormalParameter> ::= <normalFormalParameter> (`=' <expression>)?
 
 <defaultNamedParameter> ::= \gnewline{}
-  \REQUIRED? <normalFormalParameter> ((`=' | `:') <expression>)?
+  <metadata> \REQUIRED? <normalFormalParameterNoMetadata>
+  ((`=' | `:') <expression>)?
```

Comment:
If a \gnewline{} is added to line 1990 then we avoid that the rule runs into the margin:

```suggestion
  \gnewline{} ((`=' | `:') <expression>)?
```